dojo/templates/dojo/view_eng.html

```diff
@@ -649,14 +649,16 @@ <h4 class="pull-left">
                             {% for finding in findings %}
                                 <tr class="{% if finding.active %}active_finding{% else %}inactive_finding{% endif %}">
                                     <td><a href="{% url 'view_finding' finding.id %}">{{ finding.title }}</a></td>
-                                    <td>{% if finding.severity == "Critical" or finding.severity == "High" %}
-                                        <span class="text-error">
-                                    {% else %}<span>{% endif %}{{ finding.severity }}</span></td>
+                                    <td>
+                                        <span class="label severity severity-{{ finding.severity|safe }}">
```

Comment:
@patriknordlen last question I promise :joy, would you please explain this change? I understand the nav / label changes but I'm not sure why we're marking the severity as safe